chart/templates/_env/_envWorker.tpl

```diff
@@ -110,8 +110,8 @@
 # specific to 'split-descriptive-statistics'
 - name: DESCRIPTIVE_STATISTICS_HISTOGRAM_NUM_BINS
   value: {{ .Values.descriptiveStatistics.histogramNumBins | quote }}
-- name: DESCRIPTIVE_STATISTICS_MAX_PARQUET_SIZE_BYTES
-  value: {{ .Values.descriptiveStatistics.maxParquetSizeBytes | quote }}
+- name: DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES
```

Comment:
or even `MAX_SPLIT_SIZE_BYTES`? to be crystal clear that it's the size in bytes of the split's data.